### PR TITLE
Fixed the release documentation

### PR DESCRIPTION
**Pull request by bodiam**

### CODE REVIEW

File: docs/releases/2.2.0-SNAPSHOT.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Datafaker 2.1.0 (SNAPSHOT)
```
Comment (kingthorin) [inline docs/releases/2.2.0-SNAPSHOT.md:1]: 2.2.0 ? Based on file name

**Review by bodiam — COMMENTED** [review]
Fixed version

File: docs/releases/2.2.0-SNAPSHOT.md (hunk 1 above)
Comment (bodiam) [reply docs/releases/2.2.0-SNAPSHOT.md]: Good spot, fixed.

**Review by kingthorin — APPROVED** [review]
LGTM

**Pull request merged**